libnmstate/nm/applier.py

```diff
@@ -325,23 +335,40 @@ def prepare_proxy_ifaces_desired_state(ifaces_desired_state):
         new_ifaces_desired_state.append(port_iface_desired_state)
         # The "visible" slave/interface needs to point to the port profile
         iface_desired_state[MASTER_METADATA] = port_iface_desired_state[
-            Interface.NAME
+            IFACE_NAME_METADATA
         ]
         iface_desired_state[MASTER_TYPE_METADATA] = ovs.PORT_TYPE
     return new_ifaces_desired_state
 
 
 def _create_ovs_port_iface_desired_state(iface_desired_state, port_options):
+    port_name = ovs.PORT_PROFILE_PREFIX + iface_desired_state[Interface.NAME]
```

Comment:
This seems to combine a lot of changes to use the proper constants with the actual change of adding the metadata. Would be nicer to separate it. The constants change could be easily reviewed and merged.